### DISCUSSION TITLE
FTXS35KVMA working

### DISCUSSION THREAD

**Discussion by cammurray** [revk/xESP32-Faikout#359] [General]
FTXS35KVMA working fine! 

For anyone else reference here is the S21 with pin 1 at the top and pin 5 at the bottom.

![A697A90B-A00B-4F60-9FB2-B04537469B87_1_105_c](https://github.com/revk/ESP32-Faikin/assets/26195772/24c09ba5-2181-47d6-b130-8ab466bab528)

**Comment by cammurray** [comment]
Also huge kudos to the team that has made this project!!

**Comment by revk** [comment]
FYI, the clue to pin 1 is the white arrow/triangle. If that helps.

**Reply by cammurray** [reply]
@revk yeh, I found that, but if you look near pin 5 you will also see a triangle, albeit it points downwards and to the orange connector, you could think (overthink) that it might be pointing at right angles to the 5th pin on the S21. at risk of frying the faikin, I put a multimeter on it to double/triple/quadruple check.

anyway, I was surprised how easy it was, and how well it worked. after setting up MQTT to point to my mosquito instance, the daikin was autodetected in home assistant and off I went.

appears all the functions work!

**Reply by revk** [reply]
Fair enough, useful to have a clear picture.
